Core/include/Acts/Propagator/detail/SteppingHelper.hpp

```diff
@@ -33,14 +33,14 @@ namespace detail {
 template <typename stepper_t>
 Acts::Intersection3D::Status updateSingleSurfaceStatus(
     const stepper_t& stepper, typename stepper_t::State& state,
-    const Surface& surface, const BoundaryCheck& bcheck, const Logger& logger,
-    ActsScalar surfaceTolerance) {
+    const Surface& surface, Direction navDir, const BoundaryCheck& bcheck,
+    ActsScalar surfaceTolerance, const Logger& logger) {
```

Comment:
Any reason for changing the argument order here?

Reply:
right - I thought the logger is usually the last argument and since I broke the interface I could just apply a reorder as well. but its just a slight preference. I can revert that

Reply:
Up to you